[Rewrite] Function Wikis - #1 (#201)
NilPointer-Software/bdfd-wiki: src/bdscript/userServerAvatar.md

# $userServerAvatar

> Returns server avatar URL of given user ID.

Reviewer: ```suggestion
Returns the URL for the server avatar of a given user.
```

## Syntax
```
$userServerAvatar[User ID]
```

### Parameters

> - `User ID` : The user to get it's server avatar. If user doesn't have a server avatar. Then, it returns the default user avatar.

Reviewer: ```suggestion
- `User ID` : The user to get the server avatar for. If the user doesn't have a server avatar, then the default user avatar will be returned instead.
```
